vignette/montlake_seattle.html

```diff
@@ -0,0 +1,350 @@
+<!DOCTYPE html>
```

Comment:
I think this file should be deleted. Just the .Rmd is enough for pkgdown to generate the html.